.travis.yml

```diff
@@ -8,7 +8,7 @@ env:
     - TOX_ENV=py33-django16
     - TOX_ENV=py33-django17
     - TOX_ENV=py34-django16
-    - TOX_ENV=py34-django17
+    - TOX_ENV=py34-django17 DEPLOY_RELEASE=true
```

Comment:
I don't think we care: we should already have passing tests on master by the time we push a tag. I'd prefer the deploy was completely distinct from the tests, but this is an improvement in my opinion.